app/user/rest.py

```diff
@@ -461,6 +462,12 @@ def send_contact_request(user_id):
 
     if contact.is_demo_request():
         return jsonify({}), 204
+
+    try:
+        Zendesk(contact).send_ticket()
+    except Exception as e:
+        current_app.logger.exception(e)
```

Comment:
We could log the response code here as well with the logged exception.

Reply:
Zendesk error handling changed to send back the details via exception, so we should be good now